typeorm/typeorm-store/src/database.ts

```diff
@@ -54,6 +51,13 @@ export class TypeormDatabase {
         await this.con?.destroy().finally(() => this.con = undefined)
     }
 
+    protected async createDataSource(): Promise<DataSource> {
+        const cfg = createOrmConfig({projectDir: this.projectDir})
+        const connection = new DataSource(cfg)
+        await connection.initialize()
```

Comment:
I think that ".initialize()" should be called by the class to avoid a situation where the user forgets to do it himself